source/derived/mus_variable_module.f90

```diff
@@ -366,7 +367,14 @@ subroutine mus_build_varSys( varSys, solverData, schemeHeader, stencil,  &
           &                           solverData   = solverData,     &
           &                           fldLabel     = field(1)%label, &
           &                           derVarname   = derVarname      )
-        derVarPos(1)%equilFromMacro => deriveEquilPS_FromMacro
+        select case (trim(schemeHeader%order))
+        case ('first')
+          derVarPos(1)%equilFromMacro => deriveEquilPS_FromMacro
+        case ('second')
+          derVarPos(1)%equilFromMacro => deriveEquilPS2_FromMacro
+        case default
+          derVarPos(1)%equilFromMacro => deriveEquilPS2_FromMacro
+        end select
```

Comment:
Replace `schemeHeader%order`  by `schemeHeader%relaxation%variant`. The relaxation type variant will be introduced in next days. So merge this branch afterwards.

Nevertheless, set default for order when loading order from config file instead of assigning a default function pointer. In case default it is better to abort instead of assiging default to second order.

Reply:
> So merge this branch afterwards.

To clarify: we'd merge the main branch into this one and change the definitions accordingly before merging back into the main branch.